airflow/www/views.py

```diff
@@ -963,6 +963,42 @@ def run(self):
             "it should start any moment now.".format(ti))
         return redirect(origin)
 
+    @expose('/trigger')
```

Comment:
I think for now, better not to start building out an API, but continue to follow the existing pattern. If it helps, I used that code as a starting place.

Happy and excited to consider working on that API ASAP, but not for this version.